File: Java Fundamentals/Basic Sytax/Lab Tasks/Task11_MultiplicationTable2.java
import java.util.Scanner;
//100/100
public class Task11_MultiplicationTable2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);


        int number = Integer.parseInt(scanner.nextLine());
        int multiplier = Integer.parseInt(scanner.nextLine());

        if (multiplier > 10) {
            System.out.println(number + " X " + multiplier + " = " + number * multiplier);
        } else {


            for (int i = multiplier; i <= 10; i++) {

                int result = number * i;
                System.out.println(number + " X " + i + " = " + result);
            }
        }
    }
}

File: C#/C# Fundamentals/Basic Syntax, Conditional Statements & Loops/Lab Tasks/Task11_MultiplicationTable2.cs
using System;

namespace Task11_MultiplicationTable2
{
    class Task11_MultiplicationTable2
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            int multiplier = int.Parse(Console.ReadLine());

            if (multiplier > 10)
            {
                Console.WriteLine("{0} X {1} = {2}", n, multiplier, n * multiplier);
            }
            else
            {
                for (int i = multiplier; i <= 10; i++)
                {
                    Console.WriteLine("{0} X {1} = {2}", n, i, n * i);
                }
            }
        }
    }
}
